projects/editor/extensions/link/index.ts

```diff
@@ -1,8 +1,51 @@
 import {tuiParseNodeAttributes} from '@taiga-ui/editor/utils';
+import type {Editor, KeyboardShortcutCommand, Range} from '@tiptap/core';
 import {getHTMLFromFragment, markPasteRule} from '@tiptap/core';
 import {Link} from '@tiptap/extension-link';
 import {find} from 'linkifyjs';
 
+function getCurrentWordBounds(editor: Editor): Range {
+    const {state} = editor;
+    const {selection} = state;
+    const {$anchor, empty} = selection;
+
+    if (!empty) {
+        return {
+            from: selection.from,
+            to: selection.to,
+        };
+    }
+
+    if ($anchor) {
+        const {pos} = $anchor;
+        const start = $anchor.start();
+        const parent = $anchor.parent;
+        const textBefore = parent.textBetween(0, pos - start, undefined, '\uFFFC');
+        const textAfter = parent.textBetween(
+            pos - start,
+            parent.content.size,
+            undefined,
+            '\uFFFC',
+        );
+
+        const wordBefore = textBefore.replaceAll('￼', '').split(/\b/).pop();
```

Comment:
@splincode Is there a way to force ESLint to stop replacing `/\ufffc/g` with `'￼'`?

Reply:
add eslint ignore for that line

Reply:
Which rule?

Reply:
I don't know who is it


check by

```
$ npx eslint .
```